posts/2021-09-29-jcon-conference.adoc

```diff
@@ -0,0 +1,90 @@
+
+*Wednesday 6th*
+* https://jcon.sched.com/event/jzyu/exploring-stateful-microservices-built-with-open-liberty-in-kubernetes?iframe=no[Exploring Stateful Microservices built with Open Liberty in Kubernetes]
++
+CJUG's https://twitter.com/mgrygles[Mary Grygleski] will explore a tried and true, enterprise-grade, Java Open Source stack - Open Liberty in Kubernetes - which exercises and promises to fulfill the expectations of a modern server solution that epitomizes all the elements needed to build, run and deploy an efficient production system today. Mary will examine and demonstrate the fundamentals of such a system that’s built with Open Liberty in Kubernetes, including: Microservices/REST API, Concurrency, Stateful vs Stateless and Deployment.
+
+* https://jcon.sched.com/event/k3NB/restful-java-2021-edition?iframe=no[RESTful Java: 2021 edition]
++
+Open Liberty's https://twitter.com/AndrewMcCright[Andrew McCright] will compare and contrast several popular Java APIs and frameworks for creating and consuming RESTful services. He'll cover when and where some APIs make more sense than others and some basic architecture practices to facilitate porting between different frameworks.
+
+*Thursday 7th*
+* https://jcon.sched.com/event/k3MT/landscape-of-microprofile-and-jakarta-ee-tools?iframe=no[Landscape of MicroProfile and Jakarta EE]
++
+Open Liberty's https://twitter.com/yeekangc[YK Chang] and https://twitter.com/gcharters[Kathryn Kodama] will examine the tools that are available to make your life easier as you code MicroProfile and Jakarta EE-based applications. These tools help developers throughout the development lifecycle, from creating a new application, working with compatible runtimes and your favourite editor, to testing your application.
+
+* https://jcon.sched.com/event/k007/thriving-in-the-cloud-going-beyond-the-12-factors?iframe=no[Thriving in the cloud: Going beyond the 12 factors]
++
+In this session, Open Liberty's https://twitter.com/gracejansen27[Grace Jansen] will dive into the extended and updated 15 factors needed to build cloud native applications that are able to thrive in the cloud, and she'll take a look at the open source technologies and tools that can help us achieve this.
+
+* https://jcon.sched.com/event/jzzF/microprofile-40-and-beyond?iframe=no[MicroProfile 4.0 and beyond]
++
+https://twitter.com/emilyfhjiang[Emily Jiang] will discuss how MicroProfile has evolved itself to be a standard body for developing Cloud Native APIs. She'll bring you up to date with the new features in MicroProfile 4.0 with a live demo and briefly discuss what is next for this open source project. Come to this session to get yourself updated with the latest news on MicroProfile. 
```

Comment:
[Emily Jiang] will discuss how MicroProfile has evolved itself to be a standard body for developing Cloud Native APIs.
--->
[Emily Jiang] will discuss how MicroProfile has evolved to become a standard body for developing cloud-native APIs.